app/controllers/administrateurs/procedures_controller.rb

```diff
@@ -172,6 +172,9 @@ def update_monavis
     def jeton
     end
 
+    def modifications
```

Comment:
Pinaillage :

```suggestion
    def changes
```

Pour éviter le franglais ? Ou `changes_history` ? `revisions_history` ?

Reply:
les autres routes sont en français 🤷‍♂️